domain/src/main/java/com/teamagam/gimelgimel/domain/messages/poller/PolledMessagesProcessor.java

```diff
@@ -84,9 +88,17 @@ private void processMessage(Message message) {
         @Override
         public void visit(MessageGeo message) {
```

Comment:
MessageGeo is not the only message with geo-data.
There's Image, UserLocation, Sensor